### DISCUSSION TITLE
Organization-level API keys

### DISCUSSION THREAD

**Discussion by emmerich** [GitbookIO/community#30] [Feature Requests]
### What's your feature/idea?

I'd like to create API keys in my GitBook organization

### What's your desired outcome?

I currently need to create API keys on my user account. If I ever leave the organization, all the places I'm using this API key will become invalid and need to be replaced by a new API key of another org admin.

I'd like an organization API key which persists and is not tied to my membership in the org.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by pfj-joshua-golden** [comment]
Any updates on this from the Gitbook team? I was about to submit my own for this issue. The fact that API keys have to be tied to a specific user account invites a lot of issues if that user leaves a team. 

If I left my team today, so many pieces of critical automation for our document lifecycle would break. I'd love if this feature got some attention.
